Field.php

```diff
@@ -30,18 +30,60 @@ public function __construct($_width, $_height, $_maxSteps)
         $this->width = $_width;
         $this->height = $_height;
         $this->maxSteps = $_maxSteps;
-        for ($y = 0; $y < $this->height; $y++) {
-            for ($x = 0; $x < $this->width; $x++) {
+
+        $this->constructField();
+    }
+
+    /**
+     * Construct empty field
+     */
+    public function constructField()
+    {
+        for ($y = 0; $y < $this->height; $y++)
+        {
+            for ($x = 0; $x < $this->width; $x++)
+            {
                 $this->field[$x][$y] = 0;
             }
         }
     }
 
+    /**
+     * Set field's cells
+     * @param $_x int for the x-Position of the cells
+     * @param $_y int for the y-position of the cells
+     * @param $_state bool for the status of cells (dead or alive)
+     */
+    public function setField($_x, $_y, $_state)
+    {
+        if ($_x >= 0 and $_x < $this->width and $_y >= 0 and $_y < $this->height)
+        {
+            $this->field[$_x][$_y] = $_state;
+        }
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getHeight()
```

Comment:
Please rename this to `height()` and `width()` as required in our backend code style.